app/qml/Banner.qml

```diff
@@ -71,7 +72,7 @@ Rectangle {
       textItem.rightPadding: InputStyle.innerFieldMargin
       textItem.text: "<style>a:link { color: " + banner.linkColor
             + "; text-decoration: underline; }</style>" +
-            qsTr("%1<br><a href='%2'>Learn more</a>").arg(banner.text).arg(banner.link)
+            qsTr("%1<br><a href='%2' style=\"color: darkblue;\">Learn more</a>").arg(banner.text).arg(banner.link)
```

Comment:
What is the reason for changing the color? If so, can you please attach a screenshot and put this color into `InputStyle.qml`?

Reply:
MartinD mentioned in a comment on the issue that we should change the color to look more like a link than a normal text.
Here is how it looks like with the new color:
![Screenshot from 2021-11-18 22-35-08](https://user-images.githubusercontent.com/29183781/142501170-8c64efcc-cbcd-4d82-bd6e-16c239e61493.png)